sdk/core/core-http/review/core-http.api.md

```diff
@@ -907,6 +896,7 @@ export class WebResource implements WebResourceLike {
     // (undocumented)
     body?: any;
     clone(): WebResource;
+    context?: Context;
```

Comment:
`context` seems like too broad of a name. `tracingContext` perhaps?

Reply:
Mimicking the structure of `tracingOptions` has been beneficial (ie, a WebResource _is_ an OperationTracingOptions, basically) although I agree with you that it's a very generic name.

Reply:
I don't really want WebResource to have to satisfy `OperationTracingOptions` though, it doesn't try satisfy any other `OperationOptions` contract AFAIK